### DISCUSSION TITLE
When is the next version released?

### DISCUSSION THREAD

**Discussion by lortiejp** [zigpy/zigpy#937] [Q&A] [answered]
Hello! This might be a newbie question but I'm wondering when will the next zigpy version be released? I think we're on 0.43 since end of January. There is a fix that I'm specifically waiting for which is still in the dev branch and I wanted it to be in an official release to install it on my home assistant node. Maybe I'm missing some obvious information, so I'm asking the question here. :-)

Thanks!

**Comment by puddly** [comment]
@Adminiuga Do you want me to release 0.44.0?  What your usual process?  I've just been tagging `dev` in ZNP but zigpy seems to use a `master` branch as well.

**Reply by Adminiuga** [reply]
Feel free. I usually branch dev to rc, edit zigpy/__init__.py in both branches to update version then merge (no squash) rc to master and when passes, both tag master and release it in releases. It pushed to pypi upon tagging

**Comment by haaino** [comment, marked as answer]
I would also greatly appreciate a new release for the samen reason as @lortiejp ☺️

**Reply by MattWestb** [reply]
You can finding all information if you looking in the right places ;))
https://github.com/zigpy/zigpy/releases
https://github.com/home-assistant/core/pull/68921
https://github.com/home-assistant/core/pull/69099
https://github.com/home-assistant/core/releases

**Reply by lortiejp** [reply]
Thanks for making this happen. ;-)
